lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -214,7 +249,11 @@ Ufe::SceneItem::Ptr UsdHierarchy::parent() const
     // PointInstancer prim to be the "parent" of the point instance, even
     // though this isn't really true in the USD sense. This allows pick-walking
     // from point instances up to their PointInstancer.
-    return UsdSceneItem::create(fItem->path().pop(), prim().GetParent());
+    UsdPrim p = prim();
+    if (p.IsValid())
+        return UsdSceneItem::create(fItem->path().pop(), p.GetParent());
+    else
+        return Hierarchy::createItem(fItem->path().pop());
```

Comment:
Not sure why this is correct.  If we have an invalid prim, shouldn't we return a nullptr as its parent?

Reply:
I hit this case while investigating. I did not write down the exact circumstances (I think you could use the repro steps of the Maya JIRA ticket with a breakpoint in this function without my code changes.) The path parent is valid, but not the path. It was breaking checking if a node had children, IIRC.

Reply:
I can see that happening with the child of an inactive prim.  Attempting to use that prim would be invalid, its parent would not.  I suspect we are hiding a bug with this code...  We are not checking that the parent path is valid.  I would return a nullptr and see what breaks.

Reply:
The call was correct. It was trying to get the parent of the deactivated prim, IIRC. The deactivated prim has a parent, so we should not fail the call.

Reply:
But the deactivated prim is still valid, so we should never get into that branch of the conditional.

Reply:
Indeed, and yet I got into that situation, otherwise I'd not change the code :). IDK how I got there during my investigations, honestly. (and unfortunately, blame my bad memory)